faqs/gtn/get_data_upload.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How can I load data?
+area: introduction
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+1. Load by "browsing" for a local file. Best for small [datasets](https://galaxyproject.org/learn/managing-datasets/). Some servers will support load data that is 2 GB or larger.If you are having problems with this method, try FTP.
```

Comment:
`datasets` is linking to the FAQ in the Hub, which will be broken when this FAQ is migrated

Reply:
Hi @beatrizserrano. Thanks for the correction. Kindly have a look at the updates. Thanks